src/js/components/DataSearch/DataSearch.js

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from 'react';
```

Comment:
If I type '[' into DataSearch the app crashes with "Invalid regular expression: /[/: Unterminated character class"

A bit of an edge case so I'm okay if we don't handle this in this PR, but wanted to at least make note of it.

Reply:
fixed